scripts/grafana-server/custom.ini

```diff
@@ -1,2 +1,2 @@
 [feature_toggles]
-enable =
+enable = publicDashboards
```

Comment:
intentional? i've seen this file but i have no idea what it's for 😅

Reply:
Yes, as `publicDashboards` is not enabled by default, I need to turn on the feature before running the suite in order to test it. This could be removed when we turn Public Dashboards by default.